### PR TITLE
Update documentation for location table changes

### PR DESCRIPTION
**Pull request by christinach**

### CODE REVIEW

File: docs/location_changes.md [hunk 1]
```diff
@@ -1,49 +1,56 @@
...
 
-Load locally the rails server; Go to `localhost:<portnumber>/locations/holding_locations` and make sure that the locations have been updated.
+5. Copy the content from the generated .rb files into the .tmpl.rb files  
+  `cp marc_to_solr/translation_maps/location_display.rb marc_to_solr/translation_maps/location_display.rb.tmpl`
+  `cp marc_to_solr/translation_maps/locations.rb marc_to_solr/translation_maps/locations.rb.tmpl`
 
-### 2. Update Bibdata staging
-Test the updated locations in Bibdata-staging https://bibdata-staging.princeton.edu/ which is connected to the alma-sandbox;
-The locations will not be the same as in production because they are not up to date. Deploy your branch on staging and run the following steps to make sure that nothing is breaking the tables.
+6. Load locally the rails server; Go to: `localhost:<portnumber>/locations/holding_locations` and make sure that the locations have been updated.
```
Comment (sandbergja) [inline docs/location_changes.md:23]:
```suggestion
6. Load locally the rails server; Go to: `localhost:<portnumber>/locations/holding_locations` in your browser and make sure that the locations have been updated.
```

File: docs/location_changes.md [hunk 2]
```diff
@@ -1,49 +1,56 @@
...
+  `LocationDataService.delete_existing_and_repopulate`
 
-Copy the content from the generated .rb files into the .tmpl.rb files
-4.`cp marc_to_solr/translation_maps/location_display.rb marc_to_solr/translation_maps/location_display.rb.tmpl`
-`cp marc_to_solr/translation_maps/locations.rb marc_to_solr/translation_maps/locations.rb.tmpl`
+4. Generate the following files: marc_to_solr/translation_maps/location_display.rb and marc_to_solr/translation_maps/locations.rb
+  `LocationMapsGeneratorService.generate`
 
-Load locally the rails server; Go to `localhost:<portnumber>/locations/holding_locations` and make sure that the locations have been updated.
+5. Copy the content from the generated .rb files into the .tmpl.rb files
```
Comment (sandbergja) [inline docs/location_changes.md:19]:
```suggestion
5. Exit the rails console, then copy the content from the generated .rb files into the .tmpl.rb files  
```

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks, @christinach ; just two small suggestions.

File: docs/location_changes.md (hunk 2 above)
Comment (christinach) [reply docs/location_changes.md:16]: Console is not needed anymore. I added the rake task. I didn't update the wording with exit the console.

File: docs/location_changes.md [hunk 3]
```diff
@@ -1,49 +1,53 @@
...
+  `cd /opt/bibdata/current`  
+
+2. Run the following rake task to delete and repopulate the locations in the production database:  
+  `RAILS_ENV=production bundle exec rake bibdata:delete_and_repopulate_locations`
 
-Delete and repopulate the locations in the production database:
-4.`LocationDataService.delete_existing_and_repopulate`
+### Option 2:
+1. Capistrano task to connect to the production rails console:  
+  `cap production rails:console`
```
Comment (sandbergja) [inline docs/location_changes.md:48]: Now that we have the rake task (thank you!), can we just remove Option 2?

**Review by sandbergja — CHANGES_REQUESTED** [review]

**Review by sandbergja — APPROVED** [review]
Thanks @christinach !

**Pull request merged**